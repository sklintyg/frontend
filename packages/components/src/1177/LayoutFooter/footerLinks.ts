export const footerLinks = [
  {
    col: '1',
    links: [
      {
        title: 'Support',
        url: 'https://www.1177.se/e-tjanster-support',
        target: '_blank',
        requireSession: false,
      },
      {
        title: 'Tillgänglighet',
        url: 'https://www.1177.se/om-1177/1177.se/tillganglighet-pa-1177/tillganglighetsredogorelse-for-e-tjanster-pa-1177.se/',
        target: '_blank',
        requireSession: false,
      },
    ],
  },
  {
    col: '2',
    links: [
      {
        title: 'Läs mer om e-tjänster',
        url: 'https://www.1177.se/e-tjanster',
        target: '_blank',
        requireSession: false,
      },
      {
        title: 'Webbkarta',
        url: 'https://e-tjanster.1177.se/mvk/sitemap.xhtml',
        target: '_self',
        requireSession: true,
      },
    ],
  },
]
