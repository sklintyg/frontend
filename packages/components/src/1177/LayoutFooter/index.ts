export * from './LayoutFooter'
export * from './LayoutFooterLink'
export * from './LayoutFooterMobile'
