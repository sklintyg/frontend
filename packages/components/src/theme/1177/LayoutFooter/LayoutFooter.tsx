import { IDSFooter1177, IDSMobileMenu, IDSMobileMenuItem } from '@inera/ids-react'
import { AppLink } from '../../../AppLink'
import { Icon } from '../../../Icon'
import { classNames } from '../../../utils'

export function LayoutFooter({ hasSession }: { hasSession: boolean }) {
  return (
    <IDSFooter1177
      headline="1177"
      subHeadline=" - tryggt om din hälsa och vårt"
      cols={2}
      col1Size="3"
      col2Size="3"
      col1={[
        <AppLink key="support" to="https://www.1177.se/e-tjanster-support" underlined={false} external largeIcon arrow>
          Support
        </AppLink>,
        <AppLink
          key="tillgänglighet"
          to="https://www.1177.se/om-1177/1177.se/tillganglighet-pa-1177/tillganglighetsredogorelse-for-e-tjanster-pa-1177.se/"
          underlined={false}
          external
          largeIcon
          arrow
        >
          Läs mer om e-tjänster
        </AppLink>,
      ]}
      col2={[
        <AppLink key="read-more" to="https://www.1177.se/e-tjanster" underlined={false} external largeIcon arrow>
          Läs mer om e-tjänster
        </AppLink>,
        hasSession ? (
          <AppLink key="webbkarta" to="https://e-tjanster.1177.se/mvk/sitemap.xhtml" arrow>
            Webbkarta
          </AppLink>
        ) : (
          // eslint-disable-next-line react/jsx-no-useless-fragment
          <></>
        ),
      ]}
      mobileLinks={
        <>
          <AppLink to="https://www.1177.se/om-1177/1177.se/hantering-av-personuppgifter-pa-1177.se/" external>
            Behandling av personuppgifter
          </AppLink>
          <AppLink to="https://www.1177.se/om-1177/1177.se/hantering-av-kakor-cookies-pa-1177.se/" external>
            Hantering av kakor
          </AppLink>
        </>
      }
      mobileMenu={
        <IDSMobileMenu variation={1}>
          <IDSMobileMenuItem
            link={
              <a href="https://www.1177.se/e-tjanster-support" target="_blank" rel="noreferrer">
                Support
                <Icon icon="external-link" textEnd />
              </a>
            }
          />
          <IDSMobileMenuItem
            link={
              <a
                href="https://www.1177.se/om-1177/1177.se/tillganglighet-pa-1177/tillganglighetsredogorelse-for-e-tjanster-pa-1177.se/"
                target="_blank"
                rel="noreferrer"
              >
                Tillgänglighet
                <Icon icon="external-link" textEnd />
              </a>
            }
          />
          <IDSMobileMenuItem
            link={
              <a href="https://www.1177.se/e-tjanster" target="_blank" rel="noreferrer">
                Läs mer om e-tjänster
                <Icon icon="external-link" textEnd />
              </a>
            }
          />
          <IDSMobileMenuItem
            className={classNames('ids-mobile-menu-item', !hasSession && 'hidden')}
            link={
              <a href="https://e-tjanster.1177.se/mvk/sitemap.xhtml" className={classNames(!hasSession && 'hidden')}>
                Webbkarta
              </a>
            }
          />
        </IDSMobileMenu>
      }
      subFooterMobile={
        <p>
          1177 drivs av{' '}
          <AppLink to="https://www.inera.se/" colorPreset={4} small external>
            Inera AB
          </AppLink>{' '}
          på uppdrag av Sveriges regioner
        </p>
      }
      subFooterLeft={
        <p>
          1177 drivs av{' '}
          <AppLink to="https://www.inera.se/" colorPreset={4} small external>
            Inera AB
          </AppLink>{' '}
          på uppdrag av Sveriges regioner
        </p>
      }
      subFooterRight={
        <>
          <AppLink to="https://www.1177.se/om-1177/1177.se/hantering-av-personuppgifter-pa-1177.se/" external>
            Behandling av personuppgifter
          </AppLink>
          <AppLink to="https://www.1177.se/om-1177/1177.se/hantering-av-kakor-cookies-pa-1177.se/" external>
            Hantering av kakor
          </AppLink>
        </>
      }
      className="print:hidden"
    >
      <p>1177 är en tjänst från Sveriges regioner. Vi finns alltid med dig när du vill må bättre.</p>
    </IDSFooter1177>
  )
}
