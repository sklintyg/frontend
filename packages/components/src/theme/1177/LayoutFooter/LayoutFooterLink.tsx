import { IDSIconExternal } from '@frontend/ids-react-ts'
import type { ReactNode } from 'react'

export function LayoutFooterLink({ href, target, children }: { href: string; target?: string; children: ReactNode }) {
  return (
    <>
      <a className="text-white outline-white hover:underline" href={href} target={target} rel="noreferrer">
        {children}
      </a>{' '}
      {target === '_blank' && <IDSIconExternal className="relative -bottom-0.5" width="14" height="14" color="white" inline />}
    </>
  )
}
