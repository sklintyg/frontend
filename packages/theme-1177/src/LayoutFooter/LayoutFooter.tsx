import { AppLink, classNames, Icon } from '@frontend/components'
import { IDSFooter1177, IDSMobileMenu, IDSMobileMenuItem } from '@inera/ids-react'

export function LayoutFooter({ hasSession }: { hasSession: boolean }) {
  return (
    <IDSFooter1177
      headline="1177"
      subHeadline=" - tryggt om din hälsa och vård"
      cols={2}
      col1Size="3"
      col2Size="3"
      col1={[
        <AppLink key="support" to="https://www.1177.se/e-tjanster-support" external arrow>
          Support
        </AppLink>,
        <AppLink
          key="tillgänglighet"
          to="https://www.1177.se/om-1177/1177.se/tillganglighet-pa-1177/tillganglighetsredogorelse-for-e-tjanster-pa-1177.se/"
          external
          arrow
        >
          Läs mer om e-tjänster
        </AppLink>,
      ]}
      col2={[
        <AppLink key="read-more" to="https://www.1177.se/e-tjanster" external arrow>
          Läs mer om e-tjänster
        </AppLink>,
        hasSession ? (
          <AppLink key="webbkarta" to="https://e-tjanster.1177.se/mvk/sitemap.xhtml" arrow>
            Webbkarta
          </AppLink>
        ) : (
          // eslint-disable-next-line react/jsx-no-useless-fragment
          <></>
        ),
      ]}
      mobileLinks={
        <>
          <AppLink to="https://www.1177.se/om-1177/1177.se/hantering-av-personuppgifter-pa-1177.se/" underlined external>
            Behandling av personuppgifter
          </AppLink>
          <AppLink to="https://www.1177.se/om-1177/1177.se/hantering-av-kakor-cookies-pa-1177.se/" underlined external>
            Hantering av kakor
          </AppLink>
        </>
      }
      mobileMenu={
        <IDSMobileMenu variation={1}>
          <IDSMobileMenuItem
            link={
              <a href="https://www.1177.se/e-tjanster-support" target="_blank" rel="noreferrer">
                Support
                <Icon icon="external-link" textEnd />
              </a>
            }
          />
          <IDSMobileMenuItem
            link={
              <a
                href="https://www.1177.se/om-1177/1177.se/tillganglighet-pa-1177/tillganglighetsredogorelse-for-e-tjanster-pa-1177.se/"
                target="_blank"
                rel="noreferrer"
              >
                Tillgänglighet
                <Icon icon="external-link" textEnd />
              </a>
            }
          />
          <IDSMobileMenuItem
            link={
              <a href="https://www.1177.se/e-tjanster" target="_blank" rel="noreferrer">
                Läs mer om e-tjänster
                <Icon icon="external-link" textEnd />
              </a>
            }
          />
          {hasSession ? (
            <IDSMobileMenuItem
              className="ids-mobile-menu-item"
              link={
                <a href="https://e-tjanster.1177.se/mvk/sitemap.xhtml" className={classNames(!hasSession && 'hidden')}>
                  Webbkarta
                </a>
              }
            />
          ) : (
            // eslint-disable-next-line react/jsx-no-useless-fragment
            <></>
          )}
        </IDSMobileMenu>
      }
      subFooterMobile={
        <p>
          1177 drivs av{' '}
          <AppLink to="https://www.inera.se/" colorPreset={4} small underlined external>
            Inera AB
          </AppLink>{' '}
          på uppdrag av Sveriges regioner
        </p>
      }
      subFooterLeft={
        <p>
          1177 drivs av{' '}
          <AppLink to="https://www.inera.se/" colorPreset={4} small underlined external>
            Inera AB
          </AppLink>{' '}
          på uppdrag av Sveriges regioner
        </p>
      }
      subFooterRight={
        <>
          <AppLink to="https://www.1177.se/om-1177/1177.se/hantering-av-personuppgifter-pa-1177.se/" underlined external>
            Behandling av personuppgifter
          </AppLink>
          <AppLink to="https://www.1177.se/om-1177/1177.se/hantering-av-kakor-cookies-pa-1177.se/" underlined external>
            Hantering av kakor
          </AppLink>
        </>
      }
      className="print:hidden"
    >
      <p>1177 är en tjänst från Sveriges regioner. Vi finns alltid med dig när du vill må bättre.</p>
    </IDSFooter1177>
  )
}
