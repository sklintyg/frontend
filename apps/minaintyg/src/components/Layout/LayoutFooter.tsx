import { IDSFooter, IDSIconChevronBold, IDSIconExternal, IDSLink, IDSMobileMenu, IDSMobileMenuItem } from '@frontend/ids-react-ts'
import navigation from '../../data/1177-navbar-services.json'
import { resolveNavigationUrl } from '../../utils/resolveNavigationUrl'
import { LayoutFooterLink } from './LayoutFooterLink'

const footerLinks = [
  {
    col: '2',
    links: [
      ['Support', 'https://www.1177.se/e-tjanster-support', '_blank'],
      [
        'Tillgänglighet',
        'https://www.1177.se/om-1177/1177.se/tillganglighet-pa-1177/tillganglighetsredogorelse-for-e-tjanster-pa-1177.se/',
        '_blank',
      ],
    ],
  },
  {
    col: '3',
    links: [
      ['Läs mer om e-tjänster', 'https://www.1177.se/e-tjanster', '_blank'],
      ['Webbkarta', 'https://e-tjanster.1177.se/mvk/sitemap.xhtml', '_self'],
    ],
  },
]

export function LayoutFooter() {
  const startLinkItem = navigation.menu.items.find(({ name }) => name === 'Start')
  return (
    <IDSFooter type="1177" headline="1177" subheadline="– tryggt om din hälsa och vård" className="print:hidden">
      <p>1177 är en tjänst från Sveriges regioner. Vi finns alltid med dig när du vill må bättre.</p>

      {footerLinks.map(({ col, links }) =>
        links.map(([name, url, target]) => (
          <IDSLink key={`${col}-${name}`} slot={`link-col-${col}`}>
            <IDSIconChevronBold />
            <a href={url} target={target}>
              {name}
            </a>
            {target === '_blank' && <IDSIconExternal slot="append-icon" />}
          </IDSLink>
        ))
      )}

      <span slot="sub-footer-right" className="relative">
        <LayoutFooterLink href="https://www.1177.se/om-1177/1177.se/hantering-av-personuppgifter-pa-1177.se/" target="_blank">
          Behandling av personuppgifter
        </LayoutFooterLink>
      </span>
      <span slot="sub-footer-right" className="relative">
        <LayoutFooterLink href="https://www.1177.se/om-1177/1177.se/hantering-av-kakor-cookies-pa-1177.se/" target="_blank">
          Hantering av kakor
        </LayoutFooterLink>
      </span>

      {startLinkItem &&
        ['left', 'mobile'].map((slot) => (
          <span key={slot} slot={`sub-footer-${slot}`} className="relative">
            1177 drivs av{' '}
            <LayoutFooterLink href={resolveNavigationUrl(startLinkItem.url)} target="_blank">
              Inera AB
            </LayoutFooterLink>{' '}
            på uppdrag av Sveriges regioner.
          </span>
        ))}

      <IDSMobileMenu>
        {footerLinks.map(({ col, links }) =>
          links.map(([name, url, target]) => (
            <IDSMobileMenuItem key={`${col}-${name}`}>
              <a href={url} target={target} rel="noreferrer" className="relative">
                {name}
                {target === '_blank' && (
                  <IDSIconExternal
                    className="ids-ml-2 relative -bottom-0.5"
                    inline
                    color="var(--mobile-menu-item-icon_color)"
                    height="100%"
                    width="17"
                  />
                )}
              </a>
            </IDSMobileMenuItem>
          ))
        )}
      </IDSMobileMenu>

      <span slot="sub-footer-links">
        <LayoutFooterLink href="https://www.1177.se/om-1177/1177.se/hantering-av-personuppgifter-pa-1177.se/" target="_blank">
          Behandling av personuppgifter
        </LayoutFooterLink>
      </span>
      <span slot="sub-footer-links">
        <LayoutFooterLink href="https://www.1177.se/om-1177/1177.se/hantering-av-kakor-cookies-pa-1177.se/" target="_blank">
          Hantering av kakor
        </LayoutFooterLink>
      </span>
    </IDSFooter>
  )
}
