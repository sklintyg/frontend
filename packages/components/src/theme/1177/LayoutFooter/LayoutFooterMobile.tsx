import { IDSIconExternal, IDSMobileMenu, IDSMobileMenuItem } from '@inera/ids-react'
import { LayoutFooterLink } from './LayoutFooterLink'
import { footerLinks } from './footerLinks'

export function LayoutFooterMobile({ hasSession }: { hasSession: boolean }) {
  return (
    <>
      <IDSMobileMenu>
        {footerLinks.map(({ col, links }) =>
          links
            .filter(({ requireSession }) => (requireSession ? hasSession : true))
            .map(({ title, url, target }) => (
              <IDSMobileMenuItem key={`${col}-${title}`}>
                <a href={url} target={target} rel="noreferrer" className="relative">
                  {title}
                  {target === '_blank' && (
                    <IDSIconExternal
                      className="ids-ml-2 relative -bottom-0.5"
                      inline
                      color="var(--mobile-menu-item-icon_color)"
                      height="100%"
                      width="17"
                    />
                  )}
                </a>
              </IDSMobileMenuItem>
            ))
        )}
      </IDSMobileMenu>

      <span slot="sub-footer-links">
        <LayoutFooterLink href="https://www.1177.se/om-1177/1177.se/hantering-av-personuppgifter-pa-1177.se/" target="_blank">
          Behandling av personuppgifter
        </LayoutFooterLink>
      </span>
      <span slot="sub-footer-links">
        <LayoutFooterLink href="https://www.1177.se/om-1177/1177.se/hantering-av-kakor-cookies-pa-1177.se/" target="_blank">
          Hantering av kakor
        </LayoutFooterLink>
      </span>
    </>
  )
}
