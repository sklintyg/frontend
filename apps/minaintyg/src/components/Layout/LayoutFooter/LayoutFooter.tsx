import { IDSFooter, IDSIconChevronBold, IDSIconExternal, IDSLink } from '@frontend/ids-react-ts'
import footerLinks from '../../../data/footer-links.json'
import { LayoutFooterLink } from './LayoutFooterLink'
import { LayoutFooterMobile } from './LayoutFooterMobile'

export function LayoutFooter() {
  return (
    <IDSFooter type="1177" headline="1177" subheadline="– tryggt om din hälsa och vård" cols={2} className="print:hidden">
      <p>1177 är en tjänst från Sveriges regioner. Vi finns alltid med dig när du vill må bättre.</p>

      {footerLinks.map(({ col, links }) =>
        links.map(({ title, url, target }) => (
          <IDSLink key={`${col}-${title}`} slot={`link-col-${col}`}>
            <IDSIconChevronBold />
            <a href={url} target={target}>
              {title}
            </a>
            {target === '_blank' && <IDSIconExternal slot="append-icon" />}
          </IDSLink>
        ))
      )}

      <span slot="sub-footer-right" className="relative">
        <LayoutFooterLink href="https://www.1177.se/om-1177/1177.se/hantering-av-personuppgifter-pa-1177.se/" target="_blank">
          Behandling av personuppgifter
        </LayoutFooterLink>
      </span>
      <span slot="sub-footer-right" className="relative">
        <LayoutFooterLink href="https://www.1177.se/om-1177/1177.se/hantering-av-kakor-cookies-pa-1177.se/" target="_blank">
          Hantering av kakor
        </LayoutFooterLink>
      </span>

      {['left', 'mobile'].map((slot) => (
        <span key={slot} slot={`sub-footer-${slot}`} className="relative">
          1177 drivs av{' '}
          <LayoutFooterLink href="https://www.inera.se/" target="_blank">
            Inera AB
          </LayoutFooterLink>{' '}
          på uppdrag av Sveriges regioner.
        </span>
      ))}
      <LayoutFooterMobile />
    </IDSFooter>
  )
}
