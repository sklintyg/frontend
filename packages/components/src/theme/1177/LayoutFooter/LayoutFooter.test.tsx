import { render, screen } from '@testing-library/react'
import { LayoutFooter } from './LayoutFooter'

it('Should render correctly', () => {
  const { container } = render(<LayoutFooter hasSession={false} />)
  expect(container).toMatchSnapshot()
  const links = screen.getAllByRole('link', { name: 'Webbkarta' })
  expect(links).toHaveLength(2)
  links.forEach((link) => {
    expect(link).toHaveClass('hidden')
  })
})

it('Should have Webbkarta link when session is available', () => {
  render(<LayoutFooter hasSession />)
  const links = screen.getAllByRole('link', { name: 'Webbkarta' })
  expect(links).toHaveLength(2)
  links.forEach((link) => {
    expect(link).not.toHaveClass('hidden')
  })
})
