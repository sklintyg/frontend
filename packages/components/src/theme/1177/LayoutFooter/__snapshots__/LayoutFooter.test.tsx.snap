// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Should render correctly 1`] = `
<div>
  <footer
    class="ids-footer-1177 print:hidden"
  >
    <div
      class="ids-footer-1177__inner-wrapper"
    >
      <div
        class="ids-footer-1177__inner"
      >
        <div
          class="ids-footer-1177__headline-row"
        >
          <h1
            class="ids-footer-1177__headline"
          >
            1177
            <span
              class="ids-footer-1177__headline--sub"
            >
               - tryggt om din hälsa och vårt
            </span>
          </h1>
        </div>
        <div
          class="ids-footer-1177__content"
        >
          <div
            class="ids-footer-1177__text"
          >
            <p>
              1177 är en tjänst från Sveriges regioner. Vi finns alltid med dig när du vill må bättre.
            </p>
          </div>
          <div
            class="ids-footer-1177__link-col ids-footer-1177__link-col--2 ids-footer-1177__col-size-3"
          >
            <a
              class="ids-link ids-link--color-2 ids-link--block"
              href="https://www.1177.se/e-tjanster-support"
              rel="noreferrer"
              target="_blank"
            >
              <span
                class="ids-icon-arrow-right  ids-icon--text-start"
              />
              Support
              <span
                class="ids-icon-external-link  ids-icon--text-end"
              />
            </a>
            <a
              class="ids-link ids-link--color-2 ids-link--block"
              href="https://www.1177.se/om-1177/1177.se/tillganglighet-pa-1177/tillganglighetsredogorelse-for-e-tjanster-pa-1177.se/"
              rel="noreferrer"
              target="_blank"
            >
              <span
                class="ids-icon-arrow-right  ids-icon--text-start"
              />
              Tillgänglighet
              <span
                class="ids-icon-external-link  ids-icon--text-end"
              />
            </a>
          </div>
          <div
            class="ids-footer-1177__link-col ids-footer-1177__link-col--2 ids-footer-1177__col-size-3"
          >
            <a
              class="ids-link ids-link--color-2 ids-link--block"
              href="https://www.1177.se/e-tjanster"
              rel="noreferrer"
              target="_blank"
            >
              <span
                class="ids-icon-arrow-right  ids-icon--text-start"
              />
              Läs mer om e-tjänster
              <span
                class="ids-icon-external-link  ids-icon--text-end"
              />
            </a>
            <a
              class="ids-link ids-link--color-2 ids-link--block hidden"
              href="https://e-tjanster.1177.se/mvk/sitemap.xhtml"
            >
              <span
                class="ids-icon-arrow-right  ids-icon--text-start"
              />
              Webbkarta
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ids-footer-1177__sub-footer"
    >
      <div
        class="ids-footer-1177__sub-footer-container"
      >
        <div
          class="ids-footer-1177__sub-footer-row"
        >
          <div
            class="ids-footer-1177__sub-footer-left"
          >
            <p>
              1177 drivs av
               
              <a
                class="ids-link ids-link--color-4 ids-link--light ids-link--small ids-link--underlined"
                href="https://www.inera.se/"
                rel="noreferrer"
                target="_blank"
              >
                Inera AB
                <span
                  class="ids-icon-external-link  ids-icon--text-end"
                />
              </a>
               
              på uppdrag av Sveriges regioner
            </p>
          </div>
          <div
            class="ids-footer-1177__sub-footer-right"
          >
            <a
              class="ids-link ids-link--color-4 ids-link--light ids-link--small"
              href="https://www.1177.se/om-1177/1177.se/hantering-av-personuppgifter-pa-1177.se/"
              rel="noreferrer"
              target="_blank"
            >
              Behandling av personuppgifter
            </a>
            <a
              class="ids-link ids-link--color-4 ids-link--light ids-link--small"
              href="https://www.1177.se/om-1177/1177.se/hantering-av-kakor-cookies-pa-1177.se/"
              rel="noreferrer"
              target="_blank"
            >
              Hantering av kakor
            </a>
          </div>
          <div
            class="ids-footer-1177__mobile-menu"
          >
            <div
              class="ids-mobile-menu"
            >
              <div
                class="ids-mobile-menu-item"
              >
                <div
                  class="ids-mobile-menu-item__inner ids-mobile-menu-item__inner--level-1"
                >
                  <a
                    href="https://www.1177.se/e-tjanster-support"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Support
                    <span
                      class="ids-icon-external-link  ids-icon--text-end"
                    />
                  </a>
                </div>
                <div
                  class="ids-mobile-menu-item__sub-items"
                  id=":r0:"
                />
              </div>
              <div
                class="ids-mobile-menu-item"
              >
                <div
                  class="ids-mobile-menu-item__inner ids-mobile-menu-item__inner--level-1"
                >
                  <a
                    href="https://www.1177.se/om-1177/1177.se/tillganglighet-pa-1177/tillganglighetsredogorelse-for-e-tjanster-pa-1177.se/"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Tillgänglighet
                    <span
                      class="ids-icon-external-link  ids-icon--text-end"
                    />
                  </a>
                </div>
                <div
                  class="ids-mobile-menu-item__sub-items"
                  id=":r1:"
                />
              </div>
              <div
                class="ids-mobile-menu-item"
              >
                <div
                  class="ids-mobile-menu-item__inner ids-mobile-menu-item__inner--level-1"
                >
                  <a
                    href="https://www.1177.se/e-tjanster"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Läs mer om e-tjänster
                    <span
                      class="ids-icon-external-link  ids-icon--text-end"
                    />
                  </a>
                </div>
                <div
                  class="ids-mobile-menu-item__sub-items"
                  id=":r2:"
                />
              </div>
              <div
                class="ids-mobile-menu-item hidden"
              >
                <div
                  class="ids-mobile-menu-item__inner ids-mobile-menu-item__inner--level-1"
                >
                  <a
                    class="hidden"
                    href="https://e-tjanster.1177.se/mvk/sitemap.xhtml"
                  >
                    Webbkarta
                  </a>
                </div>
                <div
                  class="ids-mobile-menu-item__sub-items"
                  id=":r3:"
                />
              </div>
            </div>
          </div>
          <div
            class="ids-footer-1177__mobile-links"
          >
            <a
              class="ids-link ids-link--color-4 ids-link--light ids-link--small"
              href="https://www.1177.se/om-1177/1177.se/hantering-av-personuppgifter-pa-1177.se/"
              rel="noreferrer"
              target="_blank"
            >
              Behandling av personuppgifter
            </a>
            <a
              class="ids-link ids-link--color-4 ids-link--light ids-link--small"
              href="https://www.1177.se/om-1177/1177.se/hantering-av-kakor-cookies-pa-1177.se/"
              rel="noreferrer"
              target="_blank"
            >
              Hantering av kakor
            </a>
          </div>
        </div>
      </div>
      <div
        class="ids-footer-1177__sub-footer-mobile"
      >
        <div
          class="ids-footer-1177__sub-footer-mobile-text"
        >
          <p>
            1177 drivs av
             
            <a
              class="ids-link ids-link--color-4 ids-link--light ids-link--small ids-link--underlined"
              href="html-external-link"
            >
              Inera AB
              <span
                class="ids-icon-external-link  ids-icon--text-end"
              />
            </a>
             
            på uppdrag av Sveriges regioner
          </p>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
